modules/developers-guide/pages/cli-reference/dfx-envars.adoc

```diff
@@ -0,0 +1,41 @@
+= Environment variables
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+:sdk-short-name: DFINITY Canister SDK
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+
+You can configure certain properties for your {sdk-short-name} execution environment using environment variables.
+
+This section lists the environment variables that are currently supported with examples of how to use them.
+
+== DFX_CONFIG_ROOT
```

Comment:
Any idea why dfx chooses to call this `DFX_CONFIG_ROOT` instead of simply `DFX_ROOT`?

Is this following some larger convention?

IIUC, `$DFX_CONFIG_ROOT/.cache` and `$DFX_CONFIG_ROOT/.config` are being defined based on this variable, when set.  So, it seems strangely biased to choose a meta variable name for the root position with one root occupant (`.config`) but not the other (`.cache`).

Reply:
cc @hansl

Reply:
Its default is `$HOME`, so yes I think that's either a misnomer or we need another separate variable for CACHE.

Reply:
(I am bothered by real world https://en.wikipedia.org/wiki/Synecdoche, probably because I don't follow enough sports)